eng/Versions.props

```diff
@@ -207,9 +207,9 @@
     -->
     <Analyzer_MicrosoftCodeAnalysisCSharpVersion>3.3.1</Analyzer_MicrosoftCodeAnalysisCSharpVersion>
     <Analyzer_MicrosoftCodeAnalysisCSharpWorkspacesVersion>3.3.1</Analyzer_MicrosoftCodeAnalysisCSharpWorkspacesVersion>
-    <MicrosoftCodeAnalysisCommonVersion>4.2.0-2.22128.1</MicrosoftCodeAnalysisCommonVersion>
-    <MicrosoftCodeAnalysisCSharpVersion>4.2.0-2.22128.1</MicrosoftCodeAnalysisCSharpVersion>
-    <MicrosoftCodeAnalysisCSharpWorkspacesVersion>4.2.0-2.22128.1</MicrosoftCodeAnalysisCSharpWorkspacesVersion>
+    <MicrosoftCodeAnalysisCommonVersion>4.4.0-2.22464.20</MicrosoftCodeAnalysisCommonVersion>
+    <MicrosoftCodeAnalysisCSharpVersion>4.4.0-2.22464.20</MicrosoftCodeAnalysisCSharpVersion>
+    <MicrosoftCodeAnalysisCSharpWorkspacesVersion>4.4.0-2.22464.20</MicrosoftCodeAnalysisCSharpWorkspacesVersion>
```

Comment:
These versions aren't stable and can't be used when we hit GA. Please work w/ @jaredpar's team to understand when `4.4.0` versions will be available and on one of the feeds we build against.

Reply:
I don't think it matters that it's a preview version. We're using preview versions today - `4.2.0-2.22128.1` - and have been using that version for a long time.

These are assemblies are always private references, e.g. `PrivateAssets="All"`. They're either loaded from the SDK or VS.

As long as we aren't ahead of the version provided by the SDK or VS, which means we could be using APIs that aren't available, then I believe it's ok.

Reply:
Oh, that makes sense. If the package is consistently used in that way, non-stable dependencies are fine.